### PR TITLE
DEVDOCS-2971-add-missing-endpoints

### PR DESCRIPTION
**Pull request by traci707**
[DEVDOCS-2971](https://jira.bigcommerce.com/browse/DEVDOCS-2971)
Added missing customers/form-field-values POST endpoint.

### CODE REVIEW

**Review by bc-hnbarr — COMMENTED** [review]
I believe this looks good! Just making sure that it is intended to have as many deletions as shown. Thanks

**Review by bc-hnbarr — APPROVED** [review]

**Comment by traci707** [comment]
The swagger file had a lot of unused parameters that I removed to clean up the file.

File: reference/customers.v3.yml [hunk 1]
```diff
@@ -1556,6 +1556,47 @@ paths:
...
+                    description: The error title describing the particular error.
+                  type:
+                    type: string
+                  instance:
+                    type: string
+                  errors:
+                    type: object
+      description: Creates a list of form field values for the Customer or Customer Address object.
+      summary: Create Customer Form Field Values
+      operationId: post-customers-form-field-values
```
Comment (markcmurphy) [inline reference/customers.v3.yml:1585]:
```suggestion
      operationId: postCustomersFormFieldValues
```
I would recommend using camelcase for `operationId`

**Review by markcmurphy — APPROVED** [review]
Approving with just the note about camelcase for operationId.  The `POST` request not working on `HTTPS` means the request runner won't function and will need to be addressed at some point.

**Pull request merged**